packages/vx-shape/src/shapes/AreaClosed.js

```diff
@@ -13,7 +13,7 @@ export default function AreaClosed({
   strokeDasharray,
   strokeWidth = 2,
   stroke = 'black',
-  fill = rgba(0,0,0,0.3),
+  fill = "rgba(0,0,0,0.3)",
```

Comment:
Oh good point! Sorry about that. 

Side question: Is there an beautifier for the Airbnb JS style? Also, would anyone mind if I added the ESLint checks to the travis build?